src/controller/java/templates/ChipClusters-java.zapt

```diff
@@ -83,14 +83,12 @@ public class ChipClusters {
 
   {{#chip_client_clusters}}
   public static class {{asUpperCamelCase name}}Cluster extends BaseChipCluster {
+    public static final long CLUSTER_ID = Long.parseUnsignedLong("{{code}}");
```

Comment:
Curious, why is parsing needed here? Is it possible to just inline the value as `{{code}}L`?

```
public static final long CLUSTER_ID = {{code}}L;
```

Reply:
You could run into issues if you have a value that is out of the signed range, but that shouldn't happen for clusters since they are only 16-bit at most.

Reply:
Thought it was unsigned 64-bit for some reason - fixed.